pkg/pillar/dpcreconciler/linuxitems/route.go

```diff
@@ -44,19 +52,61 @@ func (r Route) Label() string {
 	} else {
 		dst = r.Route.Dst.String()
 	}
-	return fmt.Sprintf("IP route table %d dst %s dev %v via %v",
-		r.Table, dst, r.AdapterLL, r.Gw)
+	return fmt.Sprintf("%s route table %d dst %s dev %v via %v",
+		r.ipVersionStr(), r.Table, dst, r.AdapterLL, r.Gw)
+}
+
+func (r Route) ipVersionStr() string {
+	switch r.Family {
+	case netlink.FAMILY_V4:
+		return "IPv4"
+	case netlink.FAMILY_V6:
+		return "IPv6"
+	default:
+		return "IPv?"
```

Comment:
Could you ever get an AF_LINK or something else here?
It is easier to debug such issues if you return a string with %d with the family value.

Reply:
Within DPC Reconciler we use Route item to represent network route [copied from the main routing table to per-port RT](https://github.com/lf-edge/eve/blob/master/pkg/pillar/dpcreconciler/linux.go#L735-L755). So normally we shouldn't get routes here other than AF_INET and AF_INET6. However, we [list routes of main RT with AF_UNSPEC](https://github.com/lf-edge/eve/blob/master/pkg/pillar/netmonitor/linux.go#L335), so it probably makes sense to be prepared for something unexpected: I have added family value into the output of this function for the default value. Also Type() will return a newly defined `UnsupportedRouteTypename` in the default branch. Such route will eventually fail to be applied with [error of missing Configurator](https://github.com/lf-edge/eve/blob/master/libs/reconciler/reconciler.go#L47-L50) (rather than panicking or something like that which could be risky).